oshdb/src/test/java/org/heigit/ohsome/oshdb/osm/OSMNodeTest.java

```diff
@@ -231,16 +230,16 @@ public void testCompareToV() {
     long changeset = 4444;
     int userId = 23;
     int[] tags = new int[] {1, 1, 2, 2, 3, 3};
-    long longitude = 86809727L;
-    long latitude = 494094984L;
+    int longitude = 86809727;
+    int latitude = 494094984;
 
     OSMNode a = new OSMNode(id, version, timestamp, changeset, userId, tags, longitude, latitude);
 
     OSMNode b;
 
     b = new OSMNode(id, version + 2, timestamp, changeset, userId, tags, longitude, latitude);
 
-    assertTrue(a.compareTo(b) < 0);
+    assertEquals(true, a.compareTo(b) < 0);
```

Comment:
what was wrong with using `assertTrue` here? :zany_face:

Reply:
I'm not sure, but I think it was sonar-lint or some other code suggestions, which wanted this kind of change. I will check it which one it was.